src/main/resources/application.yaml

```diff
@@ -12,3 +12,9 @@ leanix:
     technical-user-token: ${LEANIX_TECHNICAL_USER_TOKEN}
 webhookEventService:
   waitingTime: 10000
+
+resilience4j.retry:
+  configs:
+    default:
+      maxAttempts: 3
+      waitDuration: 1000
```

Comment:
Does it mean, that it will wait 1 second after failed? the total maybe 3 seconds

Reply:
This is the waiting time in ms before retrying.